Makefile

```diff
@@ -17,7 +17,7 @@ test:
 	npm run test:ci;
 
 test-chromatic:
-	npx chromatic test run  --storybook-build-dir=storybook_dist --exit-once-uploaded --no-interactive || true
+	npx chromatic test run  --storybook-build-dir=storybook_dist --exit-once-uploaded --no-interactive || false
```

Comment:
What effect does this change have?

Reply:
If the Chromatic build fails to run, then it will fail the overall build rather than fail silently. I think this is what we want, right?

Reply:
Although you could argue that because these are not mandatory checks on the build, should it fail the build. My issue was that it was failing for a week and nobody noticed 🤦

Reply:
Ah I understand the change now, yeah I'd rather it failed if the checks don't run. I think it's okay that chromatic tests can fail, as it's an optional check, but when the tests don't even run, I'm more than happy for our build to fail.

One thing I don't quite follow is why we need the `|| false` at all? If it was just `npx chromatic test run  --storybook-build-dir=storybook_dist --exit-once-uploaded --no-interactive` wouldn't the build fail because the command would return a non 0 error code. I'm pretty sure just removing `|| false` would achieve the same thing?

Reply:
Yeah, I think we could remove the `|| false` part, I think it was a hangover from a loooooooong time ago. Let me try it out on another branch (without the upgrade to `chromatic` to see if that gives us the results we expect 🤞

Reply:
I think it can safely be removed! https://github.com/bbc/psammead/pull/3943 has failed the build at the 'ChromaticQA Test' phase 👍

Reply:
thanks for doing that @karinathomasbbc sorry if it's a bit nitpicky 🤓

Reply:
Not at all! Glad we got it sorted, and tidied up the code in the meantime 🤓

Reply:
Do we need chromatic at all?  If it stops running and nobody even notices for a week, that probably suggests it's not very useful

Reply:
It's a fair point, but a discussion we can have later - wheels are already in motion to look at reducing the usage, and we may well decide that we don't need it.